ansible_collections/arista/avd/roles/eos_designs/docs/how-to/custom-structured-configuration.md

```diff
@@ -105,59 +105,119 @@ ip name-server vrf MGMT 192.168.42.10
 ip name-server vrf MGMT 192.168.42.40
 ```
 
-#### eos_designs variables overwritten by custom_structured_configuration
+#### eos_designs variables extended by unique custom_structured_configuration when data model has no primary_key
 
 ```yaml
 ---
-# Both name_servers from eos_designs and leveraging the
-# custom_structured_configuration ONLY custom_struct will make it
-# except if using merge
+# Both name_servers from eos_designs and ip_name_servers from
+# custom_structured_configuration will make it to the intended config
 name_servers:
   - 192.168.42.10
   - 192.168.42.40
 
 custom_structured_configuration_ip_name_servers:
   - ip_address: 1.1.1.1
-    vrf: CUSTOM_STRUCT
-  - ip_address: 2.2.2.2
-    vrf: CUSTOM_STRUCT
+    vrf: EOS_CLI
+  - ip_address: 192.168.42.10
+    vrf: EOS_CLI
 ```
 
 will generate as intended config:
 
 ```eos
-ip name-server vrf CUSTOM_STRUCT 1.1.1.1
-ip name-server vrf CUSTOM_STRUCT 2.2.2.2
+ip name-server vrf EOS_CLI 1.1.1.1
```

Comment:
Let's update this IP address to a private IP.

Reply:
Disregard 1.1.1.1 is a public DNS resolver operated by Cloudflare so we can keep this!

Reply:
Just to be on a safe side - converted all Public IPs not owned by Arista to private ones